### PR TITLE
removed duplicate wildfire playbook

### PR DESCRIPTION
**Pull request by karinafishman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-32437

## Description
This PR fixes some bugs that were exists in the Detonate URL - generic v1.5 playbook:
- Added detonation with JoeSecurity vendor, instead of sub-playbook we will use it as a command.
- Deleted duplicated "Detonate URL -WildFire" playbook.
- Input of the "Detonate URL -WildFire"  playbook the was fixed, instead of using "url" we will use "upload".
- Removed the section header of the "Detonate URL - VirusTotal (API v3)". Already have it as a sub-playbook.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 1]
```diff
@@ -72,7 +72,21 @@ tasks:
...
-      - "2"
+      - "22"
+      - "12"
+      - "13"
+      - "14"
+      - "15"
+      - "16"
+      - "17"
+      - "21"
+      - "19"
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:83]: I see something weird on your server. Tasks 19 and 21 are here in the YML for the Yes condition, but on your server I don't see the Yes paths for the 2 tasks on the right side and for the task on the most left side.

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 2]
```diff
@@ -913,28 +752,111 @@ tasks:
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: 156ee7d3-6235-4011-85fe-32e8d2154de1
+    type: regular
+    task:
+      id: 156ee7d3-6235-4011-85fe-32e8d2154de1
+      version: -1
+      name: Detonate URL - JoeSecurity
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:812]: Don't we use a subplaybook for that?

File: Packs/CommonPlaybooks/doc_files/Detonate_URL_-_Generic_v1.5.png
Comment (idovandijk) [inline Packs/CommonPlaybooks/doc_files/Detonate_URL_-_Generic_v1.5.png:1]: Please update the image after the fixes

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5_README.md
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5_README.md:1]: If Joe / OPSWAT should be subplaybooks, please also update this README after the fixes

**Review by idovandijk — CHANGES_REQUESTED** [review]
Added some comments

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 2 above)
Comment (karinafishman) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:812]: We don't have a sub-playbook for that. It is deprecated and it said to use a command instead. The problem is that in this case we are not checking if the integration is working.

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml (hunk 1 above)
Comment (karinafishman) [reply Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:83]: It was there, it just was hidden. I fixed it now.

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 3]
```diff
@@ -913,28 +754,157 @@ tasks:
...
```
Comment (idovandijk) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:814]: Shoudn't this be a subplaybook?

**Review by idovandijk — CHANGES_REQUESTED** [review]
Looks good except for JoeSecurity task (see comment). Since we've made a lot of changes I suggest to compare to the playbook on master to ensure the changes are correct and that we did not introduce other issues

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml [hunk 4]
```diff
@@ -913,28 +754,157 @@ tasks:
...
+      description: Submit an URL for sandbox analysis.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic_v1.5.yml:815]:
```suggestion
      description: Submit a URL for sandbox analysis.
```

**Comment by ShirleyDenkberg** [comment]
@idovandijk Doc review completed.

**Pull request merged**